install_config/aggregate_logging.adoc

```diff
@@ -2054,6 +2054,186 @@ $ oc exec <$espod> -- es_load_kibana_ui_objects <user-name>
+  for ff in $( oc exec -c elasticsearch <espod-name> -- ls /usr/share/elasticsearch/index_patterns ) ; do
+    oc exec -c elasticsearch <es-pod-name> -- cat /usr/share/elasticsearch/index_patterns/$ff > $ff
+  done
+  break
+done
+----
++
+The index patterns download to the *_/usr/share/elasticsearch/index_patterns_*
+directory.
+
+.  If you have a separate OPS cluster, get the name of one of the es-ops 
+Elasticsearch pods:
++
+----
+$ oc get -n logging pods -l component=es-ops
+
+NAME                                           READY     STATUS    RESTARTS   AGE       IP             NODE                         NOMINATED NODE
+logging-es-ops-data-master-o7nhcbo4-5-b7stm    2/2       Running   0          38m       154.128.0.80   ip-153-12-8-6.wef.internal   <none>
+----
+
+. Load the JSON file into the es-ops Elasticsearch pod:
++
+----
+$ cat <json-file-name> | \
+oc exec -n logging -i -c elasticsearch <esops-pod-name> -- \
+    curl -s -k --cert /etc/elasticsearch/secret/admin-cert \
+    --key /etc/elasticsearch/secret/admin-key \
+    https://localhost:9200/_template/<json-file-name> -XPUT -d@- | \
+python -mjson.tool
+----
++
+[NOTE] 
+====
+The settings do not apply to existing indices. If you want to apply the settings
+to existing indices, perform a reindexing.
+====
+
+. Get the existing index pattern file from your Elasticsearch container:
++
+----
+$ mkdir index_patterns
+$ cd index_patterns
+$ oc project openshift-logging
+$ for espod in $( oc get pods -l component=es -o jsonpath='{.items[*].metadata.name}' ) ; do
+  for ff in $( oc exec -c elasticsearch <esops-pod-name> -- ls /usr/share/elasticsearch/index_patterns ) ; do
+    oc exec -c elasticsearch <esops-pod-name> -- cat /usr/share/elasticsearch/index_patterns/$ff > $ff
+  done
+  break
+done
+----
++
+The index patterns are download to the *_/usr/share/elasticsearch/index_patterns_* directory.
+
+. Identify the Kibana index patterns for which you want to add the custom fields. The index patterns 
+are listed in the Kibana UI on the left hand side of the *Management* -> *Index Patterns* page.
+
+. Edit the corresponding index pattern files to add a definition for each custom fields to the `fields` value:
++
+For example:
++
+----
+{\"count\": 0, \"name\": \"hostname\", \"searchable\": true, \"aggregatable\": true, \"readFromDocValues\": true, \"type\": \"string\", \"scripted\": false}
+----
++
+The definition must contain the `\"searchable\": true,` and `\"aggregatable\": true,` parameters to be used in visualizations.
+The data type must correspond to the Elasticsearch field definition you added above. For example, if you added a field `myfield` in Elasticsearch that is a `number` or `integer` type, you cannot add the `myfield` to Kibana as a `string` type.
+
+. Add the name of the Kibana index pattern to the index pattern files:
++
+For example, to use the *project.MYNAMESPACE.* index pattern:
++
+----
+"title": "project.MYNAMESPACE.*"
+----
+
```

Comment:
@QiaolingTang @richm 
For operations, the user would put just `"title": "operations.*"`?